pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -30,51 +30,38 @@ social:
   %br/
   = view :professional_learning_apply_banner, use_sign_up_text: true
   %br/
-  .tile-container-responsive{style: "margin-top: 20px;"}
-    .col-50
-      .tutorial-tile
-        %a{href: "/dance"}
-          %img.tutorial-tile-img{src: "/images/dance_party_2019.gif"}
-        .tutorial-info
-          %span
-            %h3.tutorial-info-h= I18n.t(:hoc_overview_dance_title)
-            .smalltext= I18n.t(:hoc_overview_dance_2019_subtitle)
-          %span
-            %a{href: "/dance"}
-              %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
-            .tutorial-info-guide
-              %a{href: "https://curriculum.code.org/hoc/plugged/8/"}
-                = I18n.t(:hoc_overview_view_teacher_guide)
 
-    .col-50
-      .tutorial-tile
-        %a{href: "/oceans"}
-          %img.tutorial-tile-img{src: "/images/fill-485x235/oceans/oceans-overview.png"}
-        .tutorial-info
-          %span
-            %h3.tutorial-info-h=hoc_s(:hoc_overview_oceans_tutorial_header)
-            .smalltext=hoc_s(:hoc_overview_oceans_tutorial_desc)
-          %span
-            %a{href: "/oceans"}
-              %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
-            .tutorial-info-guide
-              %a{href: "https://curriculum.code.org/hoc/plugged/9/"}
-                = I18n.t(:hoc_overview_view_teacher_guide)
 
-- else
-  .full-resource-block{style: "margin-top: 20px; margin-bottom: 20px;"}
-    %a{href: "/dance"}
-      %img.full-resource-image{src: "/images/dance_party_2019.gif"}
-    .tutorial-info{style: "padding: 30px;"}
-      %span
-        %h2.resource-title= I18n.t(:hoc_overview_dance_title)
-        = I18n.t(:hoc_overview_dance_2019_subtitle)
-      %span
-        %a{href: "/dance"}
-          %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
-        .tutorial-info-guide
-          %a{href: "https://curriculum.code.org/hoc/plugged/8/"}
-            = I18n.t(:hoc_overview_view_teacher_guide)
+.tile-container-responsive{style: "margin-top: 20px;"}
+  .col-50
+    .tutorial-tile
+      %a{href: "/dance"}
+        %img.tutorial-tile-img{src: "/images/dance_party_2019.gif"}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h= I18n.t(:hoc_overview_dance_title)
+          .smalltext= I18n.t(:hoc_overview_dance_2019_subtitle)
+        %span
+          %a{href: "/dance"}
+            %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
+          .tutorial-info-guide
+            %a{href: "https://curriculum.code.org/hoc/plugged/8/"}
+              = I18n.t(:hoc_overview_view_teacher_guide)
+
+  .col-50
+    .tutorial-tile
+      %a{href: "/oceans"}
+        %img.tutorial-tile-img{src: "/images/fill-485x235/oceans/oceans-overview.png"}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h=hoc_s(:hoc_overview_oceans_tutorial_header)
+          .smalltext=hoc_s(:hoc_overview_oceans_tutorial_desc)
+        %span
+          %a{href: "/oceans"}
```

Comment:
/oceans is translatable, but most of it is not translated (looks like the try now button and MS sponsorship are) on prod right now

Reply:
I'm told we are about to get a set of translations for Oceans-related content which includes strings on the site, so it should be incoming, but I will double-check.